### PR TITLE
Add SpaceFi V1 dex.trades for zkSync

### PR DESCRIPTION
**Pull request by lgingerich**

### CODE REVIEW

**Comment by lgingerich** [comment]
I'm not seeing where there would be a type conversion error in my code. Any ideas?

File: seeds/_sector/dex/spacefi_zksync_base_trades_seed.csv
Comment (Hosuke) [inline seeds/_sector/dex/spacefi_zksync_base_trades_seed.csv:1]: Seed schema is missing for this file.

**Review by Hosuke — APPROVED** [review]
LGTM.✅
Thank you @lgingerich 

[Sample spacefi_v1_zksync_base_trades](https://dune.com/queries/3566517)

File: seeds/_sector/dex/spacefi_zksync_base_trades_seed.csv
Comment (lgingerich) [reply seeds/_sector/dex/spacefi_zksync_base_trades_seed.csv:1]: Thanks I always forget about this one

Sometimes I don't get this error though even when I don't add the seed schema. Do you know why that would be?

**Review by jeff-dude — APPROVED** [review]

**Pull request merged**